gst_all_1.gst-rtsp-server has double prefix in it's .pc file

**Describe the bug**

When trying to use this package, I'm getting:

```
gcc -I... -I/nix/store/y5nlv3w8ddvy35aw3rcq8d4n5ig6ad5d-gst-rtsp-server-1.16.2-dev//nix/store/y5nlv3w8ddvy35aw3rcq8d4n5ig6ad5d-gst-rtsp-server-1.16.2-dev/include/gstreamer-1.0 -I/build/source/build -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -fPIC -MD -MQ sr
c/wfd/libwfd-server.a.p/wfd-media.c.o -MF src/wfd/libwfd-server.a.p/wfd-media.c.o.d -o src/wfd/libwfd-se
rver.a.p/wfd-media.c.o -c ../src/wfd/wfd-media.c
../src/wfd/wfd-media.c:3:10: fatal error: gst/rtsp-server/rtsp-media.h: No such file or directory
    3 | #include "gst/rtsp-server/rtsp-media.h"
```

**To Reproduce**

```
grep -R includedir $(nix-build --no-out-link -A gst_all_1.gst-rtsp-server.dev)/lib/pkgconfig
```

**Additional context**

I'm currently disabling gst-rtsp-server's split outputs in order to actually use it.

**Notify maintainers**

@bkchr 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: gst_all_1.gst-rtsp-server
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/development/libraries/gstreamer/rtsp-server/default.nix
+++ b/pkgs/development/libraries/gstreamer/rtsp-server/default.nix
@@ -20,6 +20,15 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  patches = [
+    # To use split outputs, we need this so double prefix won't be used in the
+    # pkg-config files. Hopefully, this won't be needed on the next release,
+    # _if_
+    # https://gitlab.freedesktop.org/gstreamer/gst-rtsp-server/merge_requests/1
+    # will be merged. For the current release, this merge request won't apply.
+    ./fix_pkgconfig_includedir.patch
+  ];
+
   nativeBuildInputs = [
     meson
     ninja

--- a/pkgs/development/libraries/gstreamer/rtsp-server/fix_pkgconfig_includedir.patch
+++ b/pkgs/development/libraries/gstreamer/rtsp-server/fix_pkgconfig_includedir.patch
@@ -1,0 +1,15 @@
+diff --git i/pkgconfig/meson.build w/pkgconfig/meson.build
+index 8ed8299..594cbfe 100644
+--- i/pkgconfig/meson.build
++++ w/pkgconfig/meson.build
+@@ -2,8 +2,8 @@ pkgconf = configuration_data()
+ 
+ pkgconf.set('prefix', get_option('prefix'))
+ pkgconf.set('exec_prefix', '${prefix}')
+-pkgconf.set('libdir', '${prefix}/@0@'.format(get_option('libdir')))
+-pkgconf.set('includedir', '${prefix}/@0@'.format(get_option('includedir')))
++pkgconf.set('libdir', join_paths(get_option('prefix'), get_option('libdir')))
++pkgconf.set('includedir', join_paths(get_option('prefix'), get_option('includedir')))
+ pkgconf.set('GST_API_VERSION', api_version)
+ pkgconf.set('VERSION', gst_version)
+ 

